### PR TITLE
Add initial organization charter and project governance

### PR DESCRIPTION
**Pull request by djhoese**
This PR sets up the initial governance structure for the vispy project *and* the vispy organization. It is based on GitHub's [Minimum Viable Governance](https://github.com/github/MVG) project. I haven't filled in any VisPy specific information or added any specific members to any groups. I'm hoping people will volunteer for one or both of:

1. Steering Committee
2. Maintainers

I guess I could list most of the people I consider active maintainers in the project and let you tell me if/when you don't want to be in that list.

### CODE REVIEW

File: doc/governance/MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+| **NAME**        | **Organization** |
+|-----------------|------------------|
+| Lorenzo Gaisfas | N/A              |
+| David Hoese     | N/A              |
+| Almar Klein     | N/A              |
+| Eric Larson     | N/A              |
+| Kai Mühlbauer   | N/A              |
+| Cyrille Rossant | N/A              |
+| Nicolas Rougier | N/A              |
```
Comment (djhoese) [inline doc/governance/MAINTAINERS.md:20]: @brisvag @almarklein @larsoner @kmuehlbauer @rossant @rougier I've listed all of you as maintainers to the library. If you think I've missed someone let me know. If you'd like to not be on this list, also let me know.

As you can read in the GOVERNANCE.md document, being a maintainer now doesn't mean anything different than it did before. The general idea with this governance structure is that it is all about "good faith" consensus. We can do things to this project if we think it goes along with the other maintainers' opinions/desires. The only thing that *requires* explicit consensus from a certain amount of maintainers are changes to the governance document or if anything is disagreed with.

File: doc/org/STEERING-COMMITTEE.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Committee because of your affiliation with another organization (designated
+below), you represent that you have authority to bind that organization to
+these policies.
+
+| **NAME**        | **Handle**  | **Affiliated Organization** |
+|-----------------|-------------|-----------------------------|
+| David Hoese     | @djhoese    | N/A                         |
+| Almar Klein     | @almarklein | N/A                         |
+| Cyrille Rossant | @rossant    | N/A                         |
+| Nicolas Rougier | @rougier    | N/A                         |
```
Comment (djhoese) [inline doc/org/STEERING-COMMITTEE.md:19]: @brisvag @almarklein @larsoner @kmuehlbauer @rossant @rougier I've listed me, Almar, Cyrille, and Nicolas as the members of the steering committee. The role of the committee is described in the CHARTER.md document. The only reason I didn't list everyone is that it seemed like it would make decision making harder. I don't feel that strongly about it so let me know what you think. We could always set up some kind of rotating membership of an active steering committee and inactive members. Or we could do that type of thing with the vispy project maintainers.

Thoughts?

File: doc/governance/MAINTAINERS.md (hunk 1 above)
Comment (larsoner) [reply doc/governance/MAINTAINERS.md:20]: You can remove me at this point, I'm not sure my vision of the project is clear enough anymore to make important decisions :)

Comment (djhoese) [reply doc/governance/MAINTAINERS.md]: Thanks @larsoner. I've removed you from the maintainers list. I'm sure we'll still bug you when we can.

File: doc/org/CHARTER.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
+# Charter for the [FIXME] Organization
```
Comment (brisvag) [inline doc/org/CHARTER.md:1]: fixme!

**Review by brisvag — APPROVED** [review]
LGTM!

**Comment by rougier** [comment]
Thanks for putting this up. I'm not sure to fully understand the antitrust policy because from what I've read it looks like it allows you to do pretty much anything. Probably I read it wrong. Also, is there a precise reason this is necessary?

**Comment by rougier** [comment]
Same question for trademarks: is it necessary? I guess it doesn't hurt to have it explicit but I wonder the origin of such section: does that means that some projects got problem because they did not  have such section?

**Comment by djhoese** [comment]
@rougier Thank you for bringing up these questions. You are right on the antitrust, it basically says do whatever you want. I think the main benefit to having it here is that it defines it if it ever comes up in the future. While for most people who are used to the open source scientific python ecosystem would just assume this, I don't think it hurts to state it explicitly. For the trademarks, I'm not sure if there are known instances of people doing bad stuff with trademarks, but like you said having it explicitly stated probably does more good than bad. I see the trademark statement as the "other side of the coin" to the usual request OSS projects make of "if you use our project in your research please reference the project saying that you did", where this is "if you mention our project, don't misrepresent it".

Bottom line, these documents were all included in the Minimum Viable Government project and I don't see a reason *not* to include them other than it is extra stuff to read.

**Comment by rougier** [comment]
And now the stupid question: how do I approve? I don't see any button.

**Comment by kmuehlbauer** [comment]
@rougier You would need to start a review by clicking on files on the top right of the PR I think.

**Review by kmuehlbauer — APPROVED** [review]

**Review by rougier — APPROVED** [review]

**Review by rossant — APPROVED** [review]

**Comment by djhoese** [comment]
@almarklein I know you are on paternity leave so no rush on reading through all this, but I think you're the last approval and then this can be merged.

**Review by almarklein — APPROVED** [review]

**Comment by djhoese** [comment]
Thanks everyone!
